### DISCUSSION TITLE
bun push -m "A commit message"

### DISCUSSION THREAD

**Discussion by nshiab** [oven-sh/bun#10550] [Ideas]
I don't know if it's a crazy idea, but...

I am always running these commands when working:
- git add -A
- git commit -m "My message"
- git push

Maybe we could have something like:
- bun push -m "My message"

And behind the scenes, bun would run the git commands?

Since part of Bun's mission is to simplify things, it could be a nice thing to add.

Cheers

**Comment by nexusprime-here** [comment]
Huh? 😀❓

So use package scripts

**Reply by nshiab** [reply]
You are right. 🤦
